src/Components/Admission/index.js

```diff
@@ -74,19 +73,30 @@ export default function Admission() {
 
   const [values, setValues] = useState({
     senderName: "",
-    senderEmail: "",
+    senderEmail: {
+      Cheshta: "cheshta@navgurukul.org",
+      Shahnaaz: "shahnaaz@navgurukul.org",
+      Nilam: "nilam@navgurukul.org",
+      Kitty: "kitty@navgurukul.org",
+      Rahit: "rahit@navgurukul.org",
+    },
     senderPassword: "",
     campus: "",
     name: "",
     cc: "",
     receiverEmail: "",
     date: "",
     langType: "",
+    email: "",
```

Comment:
Revert this. Change it back to senderEmail as empty string and change it based on the value selected from dropdoown.